### ISSUE TITLE
Component error when using as angular form control

### ISSUE THREAD

**Issue by Eraldo** [doxiaodong/ng2-simplemde#6]
I get an error when using `simplemde` component as an angular form control:

This works:
```
<form [formGroup]="form">
  <ion-textarea formControlName="content"></ion-textarea>
</form>
```

This does not:
```
<form [formGroup]="form">
  <simplemde formControlName="content"></simplemde>
</form>
```

Trace:
```
Error: Uncaught (in promise): TypeError: Cannot read property 'value' of undefined
TypeError: Cannot read property 'value' of undefined
    at Simplemde.writeValue (http://localhost:8100/build/vendor.js:216442:31)
    at setUpControl (http://localhost:8100/build/vendor.js:36216:27)
    at FormGroupDirective.addControl (http://localhost:8100/build/vendor.js:39184:9)
    at FormControlName._setUpControl (http://localhost:8100/build/vendor.js:39772:44)
    at FormControlName.ngOnChanges (http://localhost:8100/build/vendor.js:39690:18)
    at checkAndUpdateDirectiveInline (http://localhost:8100/build/vendor.js:11453:19)
    at checkAndUpdateNodeInline (http://localhost:8100/build/vendor.js:12952:17)
    at checkAndUpdateNode (http://localhost:8100/build/vendor.js:12891:16)
    at debugCheckAndUpdateNode (http://localhost:8100/build/vendor.js:13752:59)
    at debugCheckDirectivesFn (http://localhost:8100/build/vendor.js:13693:13)
    at c (http://localhost:8100/build/polyfills.js:3:13535)
    at Object.reject (http://localhost:8100/build/polyfills.js:3:12891)
    at Tab.NavControllerBase._fireError (http://localhost:8100/build/vendor.js:70571:16)
    at Tab.NavControllerBase._failed (http://localhost:8100/build/vendor.js:70559:14)
    at http://localhost:8100/build/vendor.js:70614:59
    at t.invoke (http://localhost:8100/build/polyfills.js:3:9283)
    at Object.onInvoke (http://localhost:8100/build/vendor.js:4510:33)
    at t.invoke (http://localhost:8100/build/polyfills.js:3:9223)
    at r.run (http://localhost:8100/build/polyfills.js:3:4452)
    at http://localhost:8100/build/polyfills.js:3:14076
```

The problematic line: `this.simplemde.value(this.value);` (https://github.com/doxiaodong/ng2-simplemde/blob/master/src/component.ts#L47)
`this.simplemde` is undefined at construction time it seems.

I think this is due to the fact that `this.simplemde` is only initialised **after** the view has initialized (`ngAfterViewInit`). And the form is initialised earlier trying to set the initial value.
 
Maybe add a check to the write method?:
```
  writeValue(v: any) {
    if (v !== this._innerValue) {
      this._innerValue = v
      if (this.simplemde && this.value != null) {
        this.simplemde.value(this.value)
      }
    }
  }
```

Any other ideas?

**Comment by doxiaodong** [comment]
I need time to check this. maybe this weekend.

**Comment by Eraldo** [comment]
Any updates yet? :)

**Comment by doxiaodong** [comment]
@Eraldo

**Comment by doxiaodong** [comment]
I didn't watch this repo util yesterday. Now it works after v1.3.1

**Comment by Eraldo** [comment]
Awesome! Thank you @doxiaodong. :)
I'll upgrade later today.

**Comment by Eraldo** [comment]
`ng2-simplemde/no-style` stopped working in the new version. Is that intentional?
My app crashes without it due to the style import. :|

**Comment by Eraldo** [comment]
The error message:
`Module '"/Users/eraldo/Documents/colegend/coapp_project/colegend/node_modules/ng2-simplemde/no-style/index"' has no exported member 'SimplemdeModule'.`

**Comment by doxiaodong** [comment]
sorry. fixed in 1.3.2

**Comment by Eraldo** [comment]
I can confirm: It works now! :)
Thank you @doxiaodong *cheer*
